### DISCUSSION TITLE
Reaching parse server from https fails with ERR_SSL_PROTOCOL_ERROR

### DISCUSSION THREAD

**Discussion by perottosson** [quasarframework/quasar#9046] [CLI - PWA mode] [answered]
I have a [parse server][1] connected with my [quasar][2] framework. I have some problems connecting to the server with https. Http works fine though. 

Running http works without any problems:

**quasar.config.js**

    // Full list of options: https://quasar.dev/quasar-cli/quasar-conf-js#Property%3A-devServer
    devServer: {
      https: false,
      port: 8080,
      open: true, // opens browser window automatically
    },

**boot/parse.js**

    import Vue from "vue";
    let parse = require("parse");
    
    parse.serverURL = "http://XXX.XXX.XXX.XXX:XXXX/parse";
    
    parse.initialize(
      "mylogin","mypassword"
    );
    
    Vue.prototype.$parse = parse;

But as soon as I switch to https (by deploying my dev server using [mkcert][3], it fails:

**quasar.config.js**

    // Full list of options: https://quasar.dev/quasar-cli/quasar-conf-js#Property%3A-devServer
    devServer: {
      https: true,
      key: fs.readFileSync('./src/certificates/localkey.pem'),
      cert: fs.readFileSync('./src/certificates/localcrt.pem'),
    
      port: 8080,
      open: true, // opens browser window automatically
    },
    

**boot/parse.js**

    import Vue from "vue";
    let parse = require("parse");
    
    parse.serverURL = "https://XXX.XXX.XXX.XXX:XXXX/parse";
    
    parse.initialize(
      "mylogin","mypassword"
    );
    
    Vue.prototype.$parse = parse;

With the errorcode:

> net::ERR_SSL_PROTOCOL_ERROR
> 
> Error: 100 XMLHttpRequest failed: "Unable to connect to the Parse API"

How can I fix this?

  [1]: https://docs.parseplatform.org/js/guide/
  [2]: https://quasar.dev/
  [3]: https://blog.filippo.io/mkcert-valid-https-certificates-for-localhost/

**Comment by hawkeye64** [comment, marked as answer]
From the quasar docs
![image](https://user-images.githubusercontent.com/10262924/116100444-273d3900-a66a-11eb-8a3c-1505da75ec9c.png)

You appear to be missing one...

**Reply by rstoenescu** [reply]
He's not only missing one, but `key` and `cert` are directly under devServer instead of devServer > https

**Reply by perottosson** [reply]
that might be but that is not the problem. I get the same error when I build it. 

```
net::ERR_SSL_PROTOCOL_ERROR

Error: 100 XMLHttpRequest failed: "Unable to connect to the Parse API"

```
